databricks_dbapi/sqlalchemy_databricks.py

```diff
@@ -0,0 +1,29 @@
+from databricks_dbapi import databricks
+from pyhive.sqlalchemy_hive import HiveDialect
+
+
+class DatabricksDialect(HiveDialect):
+    @classmethod
```

Comment:
Do we need to overwrite any HiveDialect class variables here? Should we set `name="databricks"` at least?